code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -195,7 +195,7 @@
 	return ..()
 
 /obj/structure/bed/chair/comfy/attack_hand(var/mob/user)
-	if(locked_atoms.len)
+	if(ISREALLIST(locked_atoms))
```

Comment:
This code works. It returns ..() if there is an occupant and doesn't if there isn't.